go.mod

```diff
@@ -1,15 +1,19 @@
 module github.com/ubootgame/ubootgame
 
-go 1.21.6
+go 1.20
```

Comment:
https://github.com/ebitenui/ebitenui/issues/93

didn't manage to get ebitenui working on 1.21 nor 1.22

Reply:
The fix mentioned in that thread does seem to work (tried with 1.22). We could fork it, for now...